capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -0,0 +1,169 @@
+/* eslint-disable no-unused-vars */
+
+/*
+ * Largely inspired by JS code from W3Schools: https://www.w3schools.com/howto/howto_js_autocomplete.asp
+ */
+const SEARCH_INPUT = document.getElementById('search');
+const ITEM_CLASS = 'autocomplete-items';
+const ACTIVE_CLASS = 'autocomplete-active';
+const LIST_ID = 'autocomplete-list';
+const UP_KEY = 'ArrowUp';
+const DOWN_KEY = 'ArrowDown';
+const ENTER = 'Enter';
```

Comment:
nit - match the names of the other key related constants?
```suggestion
const ENTER_KEY = 'Enter';
```